Mathlib/CategoryTheory/Comma/Over/Basic.lean

```diff
@@ -48,6 +48,9 @@ namespace Over
 
 variable {X : T}
 
+/-- The structure map of a object of `Over X`. -/
+abbrev hom {X : T} (Y : Over X) : Y.left ⟶ X := Comma.hom Y
```

Comment:
Can we have a comment explaining why this is needed, to prevent someone else "golfing" it away? I'm guessing it's so the rhs is X rather than Y.right?